e2e/tests/functional/plugins/notebook/notebookSnapshots.e2e.spec.js

```diff
@@ -90,23 +90,35 @@ test.describe('Snapshot Container tests', () => {
   test('A snapshot can be Quick Viewed from Container with 3 dot action menu', async ({ page }) => {
     await page.locator('.c-snapshot.c-ne__embed').first().getByTitle('More actions').click();
     await page.getByRole('menuitem', { name: 'Quick View' }).click();
-    await expect(page.locator('.c-overlay__outer')).toBeVisible();
+    await expect(page.getByLabel('Modal Overlay')).toBeVisible();
+    await expect(page.getByLabel('Preview Container')).toBeVisible();
+  });
+  test('A snapshot can be Viewed, Annotated, display deleted, and saved from Container with 3 dot action menu', async ({
+    page
+  }) => {
+    test.info().annotations.push({
+      type: 'issue',
+      description: 'https://github.com/nasa/openmct/issues/7552'
+    });
+    await page.getByLabel('My Items Notebook Embed').getByLabel('More actions').click();
+    await page.getByRole('menuitem', { name: 'View Snapshot' }).click();
+    await expect(page.getByRole('dialog', { name: 'Modal Overlay' })).toBeVisible();
+    await page.getByTitle('Annotate').click();
+    await expect(page.locator('#snap-annotation-canvas')).toBeVisible();
```

Comment:
can we do aria-describedby the dom id, here?

Reply:
I think that's coming from painterro itself? I don't see it in our code. still looking